### DISCUSSION TITLE
Can we add past webinar participants call?

### DISCUSSION THREAD

**Discussion by Ralpharama** [Jericho/ZoomNet#149] [Ideas]
I wonder if you might consider adding **/past_webinars/{webinarId}/participants** ?
[API ref](https://marketplace.zoom.us/docs/api-reference/zoom-api/webinars/webinarabsentees)

I see there is **/past_webinars/{WebinarUUID}/absentees** which works fine, but I need who _did_ attend, not who didn't :)

(This seems to exist for meetings, but not for webinars)

Many thanks for an excellent package otherwise!

[EDIT] Looks like zoomClient.**PastMeetings**.GetParticipantsAsync actually works for webinars, so I guess I'll use that!

**Comment by Jericho** [comment]
We sure can. There should definitely be a method to retrieve the list of registrants who attended a webinar (I think you intended to link to this documentation: https://marketplace.zoom.us/docs/api-reference/zoom-api/webinars/listwebinarparticipants).

I am very surprised to hear that getting the list of participants of a past meeting works for webinars. This is unexpected!

**Reply by Jericho** [reply]
Tracking this new feature here: https://github.com/Jericho/ZoomNet/issues/150

**Reply by Jericho** [reply]
The new feature was added in release 0.35.0

**Comment by Ralpharama** [comment]
Thanks for this, great stuff!
